Content.Server/Administration/Systems/AdminVerbSystem.Antags.cs

```diff
@@ -62,6 +64,19 @@ private void AddAntagVerbs(GetVerbsEvent<Verb> args)
         };
         args.Verbs.Add(zombie);
 
+        Verb HeadSlime = new()
+        {
+            Text = "Make HeadSlime Queen",
+            Category = VerbCategory.Antag,
+            Icon = new SpriteSpecifier.Texture(new("/Textures/Interface/Actions/headSlimeInfect.png")),
+            Act = () =>
+            {
+                _HeadSlime.HeadSlimeEntity(args.Target, null, true);
+            },
+            Impact = LogImpact.High,
+            Message = Loc.GetString("admin-verb-make-HeadSlime"),
```

Comment:
When I tried the PR locally the hover text for the make HeadSlime command just read "admin-verb-make-HeadSlime", you mighta forgotten to add this to locale